### DISCUSSION TITLE
Policy testing against AKS cluster doesn't work?

### DISCUSSION THREAD

**Discussion by DevOps-zhuang** [aquasecurity/kube-bench#765] [Questions and Help]
We are using Kube bench to test against AKS cluster, version 1.18. since it is managed cluster, we understand that the master node can't be tested. so we checked work node and policies. when checking node with running 'kube bench node', it succeeded. however, when try to test policies by running #./kube-bench --benchmark cis-1.6 run --targets policies, we just get the warning info, then we made some fix by creating new namespace and deploy workload to new namespace. rerun policies testing, it still get all warnings, so any advice? I doubt whether kube bench support policies testing against AKS? or it just support node testing against AKS? thanks.

![image](https://user-images.githubusercontent.com/54096296/98509614-79e18f00-229c-11eb-8429-28e4c5f16ca7.png)

to test policies, we use below command:
#./kube-bench --benchmark cis-1.6 run --targets policies

**Comment by mozillazg** [comment]
@DevOps-zhuang Looks like your question is similar to https://github.com/aquasecurity/kube-bench/issues/620

**Reply by DevOps-zhuang** [reply]
> @DevOps-zhuang Looks like your question is similar to #620

thanks, It seems since the rules are manually, so there is no automation testing, then just warning info. not sure it is because my target is managed K8S? can kube-bench support other K8S of the policies checking? thanks.

**Reply by mozillazg** [reply]
No, kube-bench doesn't support policy related checks not only include managed k8s but also other k8s,  the reason has been described in https://github.com/aquasecurity/kube-bench/issues/620
